systemtest/src/test/java/io/strimzi/systemtest/security/SecurityST.java

```diff
@@ -675,76 +676,111 @@ void testAutoRenewCaCertsTriggerByExpiredCertificate(ExtensionContext extensionC
         long maintenanceWindowDuration = 14;
-        maintenanceWindowStart = maintenanceWindowStart.plusMinutes(5);
+        maintenanceWindowStart = maintenanceWindowStart.plusMinutes(5).plusMinutes(Duration.ofMillis(Constants.SAFETY_RECONCILIATION_INTERVAL).toMinutes());
         final long windowStartMin = maintenanceWindowStart.getMinute();
         final long windowStopMin = windowStartMin + maintenanceWindowDuration > 59
                 ? windowStartMin + maintenanceWindowDuration - 60 : windowStartMin + maintenanceWindowDuration;
 
         String maintenanceWindowCron = "* " + windowStartMin + "-" + windowStopMin + " * * * ? *";
-        LOGGER.info("Maintenance window is: {}", maintenanceWindowCron);
+        LOGGER.info("Initial maintenanceTimeWindow is: {}", maintenanceWindowCron);
 
-        resourceManager.createResource(extensionContext, KafkaTemplates.kafkaPersistent(clusterName, 3, 1)
+        resourceManager.createResource(extensionContext, KafkaTemplates.kafkaPersistent(testStorage.getClusterName(), 3, 1)
             .editSpec()
                 .addToMaintenanceTimeWindows(maintenanceWindowCron)
+                .editKafka()
+                    .withListeners(new GenericKafkaListenerBuilder()
+                        .withName(Constants.TLS_LISTENER_DEFAULT_NAME)
+                        .withPort(9093)
+                        .withType(KafkaListenerType.INTERNAL)
+                        .withTls(true)
+                        .withAuth(new KafkaListenerAuthenticationTls())
+                    .build())
+                .endKafka()
+                .withNewClusterCa()
+                    .withRenewalDays(15)
+                    .withValidityDays(20)
+                .endClusterCa()
+                .withNewClientsCa()
+                    .withRenewalDays(15)
+                    .withValidityDays(20)
+                .endClientsCa()
             .endSpec()
             .build());
 
-
-        KafkaUser user = KafkaUserTemplates.tlsUser(clusterName, userName).build();
+        KafkaUser user = KafkaUserTemplates.tlsUser(testStorage.getClusterName(), testStorage.getUserName()).build();
 
         resourceManager.createResource(extensionContext, user);
-        resourceManager.createResource(extensionContext, KafkaTopicTemplates.topic(clusterName, topicName).build());
-        resourceManager.createResource(extensionContext, KafkaTopicTemplates.topic(clusterName, topicName).build());
-        resourceManager.createResource(extensionContext, KafkaClientsTemplates.kafkaClients(true, clusterName + "-" + Constants.KAFKA_CLIENTS, user).build());
+        resourceManager.createResource(extensionContext, KafkaTopicTemplates.topic(testStorage.getClusterName(), testStorage.getTopicName()).build());
+        resourceManager.createResource(extensionContext, KafkaTopicTemplates.topic(testStorage.getClusterName(), testStorage.getTopicName()).build());
```

Comment:
Why do you creating two same topics?

Reply:
Copy/Paste error from very beginning most probably. Fixed, thanks!